src/LinAlg/hiopMatrixRajaSparseTriplet.cpp

```diff
@@ -1163,10 +1163,10 @@ void hiopMatrixRajaSparseTriplet::print(FILE* file,
 
     if(NULL==msg) {
       if(numranks>1)
-        fprintf(file, "matrix of size %lld %lld and nonzeros %lld, printing %d elems (on rank=%d)\n", 
+        fprintf(file, "matrix of size %d %d and nonzeros %d, printing %d elems (on rank=%d)\n",
```

Comment:
I suggest replacing this with streaming operators or casting all integer arguments of `fprintf` function to `int`s. The former is probably a better long term solution. Nothing really guarantees you that e.g. `n()` will return an `int` instead of `long long`.

Reply:
We should really be using `std::format` from [fmtlib](https://github.com/fmtlib/fmt), but I'll concede to use streams for now :)